packages/governance-pool/.env.example

```diff
@@ -1,3 +1,4 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 GOERLI_URL=https://eth-goerli.g.alchemy.com/v2/<YOUR ALCHEMY KEY>
 PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
```

Comment:
Perhaps better naming here?
Ideally - not even using the PRIVATE_KEY name, but something meaningful, `DEPLOYER_PRIVATE_KEY`, or smth...

Also - I don't see if the second private key is being used anywhere or not, perhaps I missed it?